packages/ocaml-protoc-plugin/ocaml-protoc-plugin.4.3.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+maintainer: "Issuu"
+authors: "Anders Fugmann <anders@fugmann.net>"
+license: "APACHE-2.0"
+homepage: "https://github.com/issuu/ocaml-protoc-plugin"
+dev-repo: "git+https://github.com/issuu/ocaml-protoc-plugin"
+bug-reports: "https://github.com/issuu/ocaml-protoc-plugin/issues"
+doc: "https://issuu.github.io/ocaml-protoc-plugin/"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test & arch != "arm32" & arch != "x86_32"}
+]
+
+depends: [
+  "conf-protoc" {>= "1.0.0"}
+  "dune" {>= "3.2"}
+  "ocaml" {>= "4.08.0"}
+  "ppx_expect" {with-test}
+  "ppx_inline_test" {with-test}
+  "ppx_deriving" {with-test}
+  "conf-pkg-config" {build}
+  "dune-configurator"
+]
+
+
```

Comment:
```suggestion
available: arch != "s390x"
```

Reply:
If it is broken there, I suggest to mark it unavailable. Or it is a minor incompatibility and most functionalities are expected to work?

Reply:
Its a real bug on all big endian architectures. which has not been discovered due to lack of testing on said architectures. 
I already created a fix for this, and its pending review. I will close this PR, and create a new release once the fix is in.